### PR TITLE
Fix unclosed backquotes in Code.Fragment and Macro

### PR DESCRIPTION
**Pull request by wingyplus**
Found from working #12038. Fixes 2 ex_doc warning. Here is build log https://github.com/wingyplus/elixir/runs/7727863757?check_suite_focus=true#step:9:111

### CODE REVIEW

File: lib/elixir/lib/macro.ex [hunk 1]
```diff
@@ -1969,7 +1969,7 @@ defmodule Macro do
   def quoted_literal?(term), do: is_atom(term) or is_number(term) or is_binary(term)
 
   @doc """
-  Expands a `quoted_literal` with the given `
+  Expands a `quoted_literal` with the given ```
```
Comment (wingyplus) [inline lib/elixir/lib/macro.ex:1972]: This is still warning

Comment (wingyplus) [reply lib/elixir/lib/macro.ex]: Fixed.

Comment (wingyplus) [reply lib/elixir/lib/macro.ex]: Before fix:

Desktop Screenshot 2022 08 08 - 21 53 44 45

After fixed:

Desktop Screenshot 2022 08 08 - 21 52 18 85

Comment (wingyplus) [reply lib/elixir/lib/macro.ex]: But it should `with the given AST` not backquote?

File: lib/elixir/lib/macro.ex [hunk 2]
```diff
@@ -1969,7 +1969,7 @@ defmodule Macro do
   def quoted_literal?(term), do: is_atom(term) or is_number(term) or is_binary(term)
 
   @doc """
-  Expands a `quoted_literal` with the given `
+  Expands a `quoted_literal` with the given \\`
```
Comment (josevalim) [inline lib/elixir/lib/macro.ex:1972]:
```suggestion
  Expands the `ast` representing a quoted literal within the given `env`.
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**